adafruit_mcp9600.py

```diff
@@ -82,6 +193,77 @@ def __init__(self, i2c, address=_DEFAULT_ADDRESS, tctype="K", tcfilter=0):
+                                       For rising temperatures, the hysteresis is below alert limit.
+                                       For falling temperatures, the hysteresis is above alert
+                                       limit. See data-sheet for further information.
+        :param alert_temp_direction: The direction the temperature must change to trigger the alert.
+                                     Options are ``RISING`` (heating up) or ``FALLING`` (cooling
+                                     down).
+        :param alert_mode: The alert mode. Options are ``COMPARATOR`` or ``INTERRUPT``. If setting
+                           mode to ``INTERRUPT``, use ``alert_interrupt_clear`` to clear the
+                           interrupt flag.
+        :param alert_state: Alert pin output state. Options are ``ACTIVE_HIGH`` or ``ACTIVE_LOW``.
+
+
+        For example, to configure alert 1:
+
+        .. code-block:: python
+
+            import board
+            import busio
+            import digitalio
+            import adafruit_mcp9600
+
+            i2c = busio.I2C(board.SCL, board.SDA, frequency=100000)
+            mcp = adafruit_mcp9600.MCP9600(i2c)
+            alert_1 = digitalio.DigitalInOut(board.D5)
+            alert_1.switch_to_input()
+
+            mcp.burst_mode_samples = mcp.BURST_SAMPLES_1
+            mcp.ambient_resolution = mcp.AMBIENT_RESOLUTION_0_25
+            mcp.alert_config(alert_number=1, alert_temp_source=mcp.THERMOCOUPLE,
+                             alert_temp_limit=25, alert_hysteresis=0,
+                             alert_temp_direction=mcp.RISING, alert_mode=mcp.COMPARATOR,
+                             alert_state=mcp.ACTIVE_LOW)
+
+        """
+        if alert_number not in (1, 2, 3, 4):
+            raise ValueError("Alert pin number must be 1-4.")
+        if not 0 <= alert_hysteresis < 256:
+            raise ValueError("Hysteresis value must be 0-255.")
+        setattr(self, '_alert_%d_monitor' % alert_number, alert_temp_source)
+        setattr(self, '_alert_%d_temperature_limit' % alert_number, int(alert_temp_limit / 0.0625))
+        setattr(self, '_alert_%d_hysteresis' % alert_number, alert_hysteresis)
+        setattr(self, '_alert_%d_temp_direction' % alert_number, alert_temp_direction)
+        setattr(self, '_alert_%d_mode' % alert_number, alert_mode)
+        setattr(self, '_alert_%d_state' % alert_number, alert_state)
+        setattr(self, '_alert_%d_enable' % alert_number, True)
+
+    def alert_disable(self, alert_number):
+        """Configuring an alert using ``alert_config()`` enables the specified alert by default.
+        Use ``alert_disable`` to disable an alert pin."""
+        setattr(self, '_alert_%d_enable' % alert_number, False)
+
+    def alert_interrupt_clear(self, alert_number, interrupt_clear=True):
+        """Setting ``interrupt_clear==True`` clears the interrupt flag."""
```

Comment:
Turns off the alert flag in the MCP9600, and clears the pin state (not used if the alert is in comparator mode)

:param int alert_number: The alert pin number. Must be 1-4.
:param bool interrupt_clear The bit to write the interrupt state flag